src/transformers/pipelines/automatic_speech_recognition.py

```diff
@@ -328,6 +328,9 @@ def postprocess(self, model_outputs, decoder_kwargs: Optional[Dict] = None, retu
                 # because of padding, that's why
                 # we need to reconstruct this information
                 # This won't work with left padding (which doesn't exist right now)
+
+                # Hack
+                right = 0 if idx == len(model_outputs) - 1 else right
```

Comment:
@Narsil - just added this hack here to show where the error comes from. Don't think that this is the best solution, but it would probably work. Could you maybe take a look ?